api/MouseEvent.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseEvent",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": true
           },
           "chrome": {
-            "version_added": null
+            "version_added": true
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": true
```

Comment:
The MDN table has it listed as false for Chrome for Android, I'm not sure what mouse support is like for android browsers? Ditto for Webview Android.

Reply:
I marked it as `true` because I was thinking something like: "How can basic support be false when sub-feature support is true?"
Waiting on additional input for how to proceed.